design/run_and_collect_data.md

```diff
@@ -0,0 +1,27 @@
+#run.py proposal
```

Comment:
yep.  I've added the proposed .csv to the design folder for now.  You can amend it to let me know if you want to track more.  I was thinking if we are going to be running each heuristic multiple times for a test_case, maybe we should track averages as well as best.  that shouldn't be too hard to add though